gem/rbi-central.gemspec

```diff
@@ -32,6 +32,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency("spoom", ">= 1.1.12")
   spec.add_dependency("tapioca", ">= 0.9.2")
   spec.add_dependency("thor", ">= 1.2.1")
+  spec.add_development_dependency("debug")
```

Comment:
Same, I think this only makes sense inside `gem/`

Reply:
Do you mean in `gem/Gemfile` rather than the `gemspec`?

Reply:
My bad I didn't read the filename properly 🤦 

Here's good 👍